package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/generator-add-action-generic",
-  "version": "0.2.3",
+  "version": "0.2.4",
```

Comment:
everything else looks good, can we revert this version change though? it is automated by our publish action

Reply:
@shazron , ^^^ done